app/site/_includes/tag.liquid

```diff
@@ -0,0 +1,46 @@
+<script>
+    document.addEventListener('DOMContentLoaded', function() {
+      // Function to set the background color based on project type
+      
+        function setTagColor(projType) {
+            let color;
+            switch (projType) {
+            case 'Tools':
+                color = 'green';
+                break;
+            case 'APIs':
+                color = 'red';
+                break;
+            case 'Apps':
+                color = 'blue';
+                break;
+            case 'Data':
+                color = 'orange';
+                break;
+            case 'Docs':
+                color = 'mint';
+                break;
+            case 'Libraries':
+                color = 'magenta';
+                break;
+            case 'Standards':
+                color = 'gold';
+                break;
+            default:
+                color = '#1a4480';
+            }
```

Comment:
It might be easier to use an object literal here since you're dealing with all strings. So something like

```javascript
const TAG_COLORS = {
  "Tools": "green",
  "APIs": "red",
  // ...
}

const color = TAG_COLORS[projType] || "#1a4480"
```